.golangci.yaml

```
@@ -0,0 +1,52 @@
+run:
+  timeout: 10m
+  skip-dirs:
+    - docs
+    - scripts
+
+linters-settings:
+  gofumpt:
+    extra-rules: true
+  gci:
+    sections:
+      - standard
+      - default
+      - prefix(github.com/virtualization/)
```

```suggestion
      - prefix(github.com/deckhouse/virtualization/)
```
I think github.com/virtualization is not what we want